scripts/build.py

```diff
@@ -431,4 +566,7 @@ def generate_build_directory():
 
 
 if __name__ == '__main__':
-    generate_build_directory()
+    if feconf.FORCE_PROD_MODE:
```

Comment:
Better to have this file be independent of feconf (and anything in the oppia codebase). Can you pass in the prod_env flag from the shell script instead and branch based on that?

Also make sure to either have a default, or force the user to specify (e.g. using argparse). Print enough logging info to the terminal so that the user can see what mode the build script is running in.

Reply:
Done. Default is dev mode.